exercises/strings/capitalize/capitalize.js

```diff
@@ -12,10 +12,15 @@
 
 function capitalize(num) {
   // This is your job. :)
+  let firstcharactercap= num.slice(0,1).toUpperCase();
```

Comment:
The convention in JavaScript is to name variables using what's called "camel case". So rather than writing `first_character` or `firstcharacter` you'd write `firstCharacter`. Try to follow that convention; it makes it easier to spot typos and for other JS programmers to understand your code at a glance.